packages/NodeNameResolver/NodeNameResolver.php

```diff
@@ -63,14 +63,35 @@ public function isName(Node | array $node, string $name, bool $strict = false):
         $nodes = is_array($node) ? $node : [$node];
 
         foreach ($nodes as $node) {
-            if ($this->isSingleName($node, $name, $strict)) {
+            if ($this->isSingleName($node, $name)) {
                 return true;
             }
         }
 
         return false;
     }
 
+    public function isNameExactly(Node $node, string $name): bool
```

Comment:
Not sure if "exactly" is common language in PHP world. "Insenstive" is standard name for "exact" check.

Reply:
Indeed, the `isNameCaseSensitive()` suggest something different :).
I would go with `isCaseSensitiveName()`. 

Identical/equal are kind of ambiguous too. Never know which is which untill I run the PHPUnit test :D

Reply:
Good point. Fixed.